syntax = "proto3";
package base;
import "base/hash.proto";

message SHARE_DMHY_ORG_TRACKER {
    repeated SHARE_DMHY_ORG_TRACKER_CONFIG configs = 1;
}

message SHARE_DMHY_ORG_TRACKER_CONFIG {
    string key = 1;
    uint64 bangumi_id = 2;
    string rss_url = 3;
    HASH latest_resp_hash = 4;
    // "2024-04-06T12:10:27.091865Z"
    string latest_update_time = 5;
}